File: Java/day8.java
import java.util.Scanner;

public class Rootzzz {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a: ");
        double a = sc.nextDouble();
        System.out.print("Enter b: ");
        double b = sc.nextDouble();
        System.out.print("Enter c: ");
        double c = sc.nextDouble();

        double d = b * b - 4 * a * c;

        if (d> 0) {
            double r1 = (-b + Math.sqrt(d)) / (2 * a);
            double r2 = (-b - Math.sqrt(d)) / (2 * a);
            System.out.println("Two real roots: " + r1 + " and " + r2);
        } else if (d == 0) {
            double r = -b / (2 * a);
            System.out.println("One real root: " + r);
        } else {
            double r = -b / (2 * a);
            double img = Math.sqrt(-d) / (2 * a);
            System.out.println("Two complex roots: " + r+ " + " + img + "i and " + r + " - " + img + "i");
        }

        sc.close();
    }
}

File: Python/day8.py
import math

def rootsof(a, b, c):
    dis = b**2 - 4*a*c
    
    if dis > 0:
        root1 = (-b + math.sqrt(dis)) / (2*a)
        root2 = (-b - math.sqrt(dis)) / (2*a)
        print(f"Two real roots: {root1} and {root2}")
    elif dis == 0:
        root = -b / (2*a)
        print(f"One real root: {root}")
    else:
        real = -b / (2*a)
        imag = math.sqrt(-dis) / (2*a)
        print(f"Two complex roots: {real} + {imag}i and {real} - {imag}i")

a = float(input("Enter a: "))
b = float(input("Enter b: "))
c = float(input("Enter c: "))
rootsof(a, b, c)
